.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🚀 Feature request
+    url: https://pact.canny.io
+    about: The Canny board to send us feature requests, vote and measure the interest of users. Useful to submit a feature request when you have an idea but no concrete API design proposal.
+  - name: ❓ Simple question - Slack chat
+    url: https://slack.pact.io
+    about: If you have a simple question, or want to discuss a feature request, join our Slack chat.
+  - name: ❓ Simple question - Stack Overflow
+    url: https://stackoverflow.com/questions/tagged/pact
+    about: The GitHub issue tracker is not for technical support. Please use Stack Overflow, and ask the community for help.
+  - name: ❓ Advanced question - GitHub Discussions
+    url: https://github.com/pact-foundation/pact-python/discussions
```

Comment:
ahhh nice we have the advanced question here, which we can reference further up

https://github.com/pact-foundation/pact-python/pull/390/files#diff-45b5634e925b86895feee745ec5650893bae0d56e11b379745e506fd9a6b81bdR16

Reply:
Given the user filing a bug will have to see this first, I think it is a bit redundant to refer back to this once they have selected bug. The comment about asking on Slack is more to address the "I'm not sure whether this is my bug, or Pact Python's bug" and where they can get help.

Reply:
ahhh yes, they see this as the first page, prior to selecting their option from this config 👍🏾